spec/features/edit_profile_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'editing your user profile', type: :feature do
+  context 'with a user' do
+    let(:user) { create(:user, email: 'test@stanford.edu') }
+
+    before do
+      login_as(user, scope: :user)
+    end
+
+    it 'requires current password to change password' do
+      visit '/users/edit'
+      fill_in 'user_password', with: '123'
+      click_on 'Change Password'
+      expect(page).to have_content "Current password can't be blank"
```

Comment:
Single quotes for consistency:

```suggestion
      expect(page).to have_content 'Current password can't be blank'
```